aws/ec2.go

```diff
@@ -13,15 +13,15 @@ import (
 
 func portHash(username string) int64 {
```

Comment:
Can we change the name of this from `portHash` to something like `ruleIDHash` or something?  `portHash` seems misleading as the result is not used as a port at all.

Reply:
see below commit

Reply:
I can see James' comment has been addressed so going to approve.